web/themes/custom/bixal_uswds/src/sass/custom-components/components.scss

```diff
@@ -4,3 +4,18 @@
 .component__item {
   @include u-margin-bottom(5);
 }
+
+h1.blog-title {
```

Comment:
This seems like it will make it not maintainable between storybook & Drupal.
If we need to have a heading that has space underneath it with with a colored line, can we just add a class and allow that to be added to any header tag?

Reply:
Messaged you about this change and removed this part